src/test/bitcoin-util-test.py

```diff
@@ -6,8 +6,25 @@
 import os
 import bctest
 import buildenv
+import sys
```

Comment:
Ooops. Nothing now. I was originally using `sys.argv`, but then changed to use `argparse`. I'll remove it and squash the commits.